.github/contributing.md

```diff
@@ -29,21 +38,15 @@ technically stable, they tend to not work reliably with Minecraft 1.9+.
 Second, test without addons. No, seriously; unless you're an addon developer,
 test without plugins that hook to Skript before reporting anything. We can't
 help you with addon issues here, unless we get a lot of technical information
-about the addon in question. Usually only developers of them know addons'
-Java code well enough.
+about the addon in question. Issues that are not tested without addons are
+likely to be ignored by the core team.
 
 If the issue still persists, search the issue tracker for similar
 errors and check if your issue might have been already reported.
 Only if you can't find anything, open a new issue.
 
-Now, what would you fill to the issue report?
-* A useful title (tl;dr for us busy and/or lazy developers)
-* Description of the issue, aka what does *not* work
-* Any error messages in the console (please use a paste service for lenghty errors)
-* Whether you tested without addons or not
-  - *Sometimes* we do not ignore people who are testing with addons
-  - Testing without is still better, unless you're an addon developer
-* Any additional information you think would be helpful
+When opening an bug template, pick a template for a bug and fill it.
```

Comment:
Shouldn't this be "When opening a bug report, ..."

Reply:
Nope, github has a new issue template system. You'll see what it means when this is merged. However, the English of this sentence can be improved by changing it to: `When opening an issue, please pick the template most appropriate and fill out the template completely`

Reply:
@JRoy no, you both are right - we're opening issues (bug reports) from the templates, but definitely we aren't opening hug

Reply:
It is a typo, not intended.